playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -84,6 +85,39 @@ def silence_alert(alert: PrometheusKubernetesAlert, params: SilenceAlertParams):
     alert.stop_processing = True
 
 
+class StatusSilencerParams(ActionParams):
+    """
+    :var include: If available, will stop processing unless the pod status is in the include list
+    :var exclude: If available, will stop processing if the pod status is in the exclude list
+
+    :example include: ["Pending"]
+    :example exclude: ["Evicted"]
```

Comment:
the status's for phase are only: pending, running, succeded, failed, unknown
Evicted isn't possible for phase
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#pod-phase

Reply:
You have a point.
I'm not sure "Evicted" is a phase.
Going to change the example